### ISSUE TITLE
Don't support yarn installed from system package manager

### ISSUE THREAD

**Issue by ungarscool1** [alexindigo/install-peers#6]
**Describe the bug**
I have installed yarn with my system package manager and when I install `install-peers` as dev dependency. The package won't find yarn binary.

**To Reproduce**
1. Install yarn with apt
2. Run `git clone <a repo with install-peers in dev-dep>`
3. Run `yarn`
4. Peer dependencies are not installed

Same behavior with `node:14-alpine` docker container.

**Environment variables**
Docker container:
```
npm_package_devDependencies__types_node=^17.0.29
npm_package_devDependencies_jest=^27.4.7
npm_package_peerDependencies__dazn_lambda_powertools_logger=^1.28.1
npm_config_version_commit_hooks=true
npm_config_user_agent=yarn/1.22.17 npm/? node/v14.19.1 linux x64
NODE_VERSION=14.19.1
npm_config_bin_links=true
HOSTNAME=bb1a10b1fd7e
YARN_VERSION=1.22.17
npm_node_execpath=/usr/local/bin/node
npm_config_init_version=1.0.0
SHLVL=4
HOME=/root
OLDPWD=/
npm_package_devDependencies__tsconfig_node14=^1.0.1
npm_config_init_license=MIT
YARN_WRAP_OUTPUT=false
npm_config_version_tag_prefix=v
npm_package_devDependencies_typescript=^4.6.3
npm_package_devDependencies_aws_sdk_mock=^5.6.2
npm_package_devDependencies__aws_sdk_types=^3.78.0
npm_package_scripts_test_coverage_junit=jest --ci --collect-coverage --reporters=jest-junit
npm_package_devDependencies_ts_jest=^27.1.4
npm_package_devDependencies__types_jest=^27.4.1
npm_package_private=true
npm_config_registry=https://registry.yarnpkg.com
TERM=xterm
npm_config_ignore_scripts=
npm_config_version=1.22.17
npm_package_dependencies_axios=^0.27.1
npm_package_author_email=<REDACTED>
PATH=/tmp/yarn--1651594327499-0.4972750684203364:/project/node_modules/.bin:/usr/local/share/.config/yarn/link/node_modules/.bin:/usr/local/libexec/lib/node_modules/npm/bin/node-gyp-bin:/usr/local/lib/node_modules/npm/bin/node-gyp-bin:/usr/local/bin/node_modules/npm/bin/node-gyp-bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
NODE=/usr/local/bin/node
npm_package_name=<REDACTED>
npm_package_devDependencies_eslint=^8.14.0
npm_lifecycle_script=env
npm_package_devDependencies_install_peers=^1.0.3
npm_package_main=index.js
npm_package_scripts_test=jest
npm_config_version_git_message=v%s
npm_lifecycle_event=oui
npm_package_devDependencies_jest_junit=^13.2.0
npm_package_version=1.0.0
npm_config_argv={"remain":[],"cooked":["run","oui"],"original":["oui"]}
npm_package_devDependencies__types_aws_lambda=^8.10.95
npm_package_scripts_build=<REDACTED>
npm_package_scripts_oui=env
npm_package_scripts_test_coverage=jest --ci --coverage
npm_config_version_git_tag=true
npm_config_version_git_sign=
npm_package_peerDependencies_aws_sdk=^2.1122.0
npm_package_license=Apache-2.0
npm_config_strict_ssl=true
PWD=/project
npm_execpath=/opt/yarn-v1.22.17/bin/yarn.js
npm_package_author_name=<REDACTED>
npm_config_save_prefix=^
npm_config_ignore_optional=
INIT_CWD=/project
```

My computer:
```
npm_package_devDependencies__types_node=^17.0.29
npm_package_devDependencies_jest=^27.4.7
npm_package_peerDependencies__dazn_lambda_powertools_logger=^1.28.1
npm_config_version_commit_hooks=true
npm_config_user_agent=yarn/1.22.18 npm/? node/v14.18.2 linux x64
npm_config_bin_links=true
npm_node_execpath=/home/leo/.nvm/versions/node/v14.18.2/bin/node
npm_config_init_version=1.0.0
NVM_BIN=/home/leo/.nvm/versions/node/v14.18.2/bin
TERM_PROGRAM_VERSION=1.66.2
npm_package_devDependencies__tsconfig_node14=^1.0.1
NVM_INC=/home/leo/.nvm/versions/node/v14.18.2/include/node
npm_config_init_license=MIT
VSCODE_GIT_ASKPASS_MAIN=/home/leo/.vscode-server/bin/dfd34e8260c270da74b5c2d86d61aee4b6d56977/extensions/git/dist/askpass-main.js
YARN_WRAP_OUTPUT=false
npm_config_version_tag_prefix=v
VSCODE_GIT_ASKPASS_NODE=/home/leo/.vscode-server/bin/dfd34e8260c270da74b5c2d86d61aee4b6d56977/node
npm_package_devDependencies_typescript=^4.6.3
NVM_DIR=/home/leo/.nvm
npm_package_devDependencies_aws_sdk_mock=^5.6.2
npm_package_devDependencies__aws_sdk_types=^3.78.0
npm_package_scripts_test_coverage_junit=jest --ci --collect-coverage --reporters=jest-junit
_=/home/leo/.nvm/versions/node/v14.18.2/bin/yarn
project=/home/leo/project
npm_package_devDependencies_ts_jest=^27.1.4
npm_package_devDependencies__types_jest=^27.4.1
npm_package_private=true
npm_config_registry=https://registry.yarnpkg.com
npm_config_ignore_scripts=
npm_package_dependencies_axios=^0.27.1
npm_package_author_email=<REDACTED>
PATH=/tmp/yarn--1651594504335-0.7323585250548508:/home/leo/project/node_modules/.bin:/home/leo/.config/yarn/link/node_modules/.bin:/home/leo/.yarn/bin:/home/leo/.nvm/versions/node/v14.18.2/libexec/lib/node_modules/npm/bin/node-gyp-bin:/home/leo/.nvm/versions/node/v14.18.2/lib/node_modules/npm/bin/node-gyp-bin:/home/leo/.nvm/versions/node/v14.18.2/bin/node_modules/npm/bin/node-gyp-bin:/home/leo/.vscode-server/bin/dfd34e8260c270da74b5c2d86d61aee4b6d56977/bin/remote-cli:/home/leo/.nvm/versions/node/v14.18.2/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/lib/wsl/lib:/mnt/c/Windows/system32:/mnt/c/Windows:/mnt/c/Windows/System32/Wbem:/mnt/c/Windows/System32/WindowsPowerShell/v1.0/:/mnt/c/Windows/System32/OpenSSH/:/mnt/c/Program Files/Git/cmd:/mnt/c/HashiCorp/Vagrant/bin:/mnt/c/Program Files/Docker/Docker/resources/bin:/mnt/c/ProgramData/DockerDesktop/version-bin:/mnt/c/Program Files (x86)/Intel/Intel(R) Management Engine Components/DAL:/mnt/c/Program Files/Intel/Intel(R) Management Engine Components/DAL:/mnt/c/Program Files/WireGuard/:/mnt/c/Users/LéoGodard/AppData/Local/Microsoft/WindowsApps:/mnt/c/Users/LéoGodard/AppData/Local/Programs/Microsoft VS Code/bin:/mnt/c/Users/LéoGodard/Documents/bin
NODE=/home/leo/.nvm/versions/node/v14.18.2/bin/node
npm_package_name=<REDACTED>
npm_package_devDependencies_eslint=^8.14.0
npm_lifecycle_script=env
npm_package_devDependencies_install_peers=^1.0.3
npm_package_main=index.js
npm_package_scripts_test=jest
npm_config_version_git_message=v%s
npm_lifecycle_event=oui
npm_package_devDependencies_jest_junit=^13.2.0
npm_package_version=1.0.0
npm_config_argv={"remain":[],"cooked":["run","oui"],"original":["oui"]}
npm_package_devDependencies__types_aws_lambda=^8.10.95
npm_package_scripts_build=<REDACTED>
npm_package_scripts_oui=env
npm_package_scripts_test_coverage=jest --ci --coverage
npm_config_version_git_tag=true
npm_config_version_git_sign=
npm_package_peerDependencies_aws_sdk=^2.1122.0
npm_package_license=Apache-2.0
npm_config_strict_ssl=true
PWD=/home/leo/project
npm_execpath=/usr/share/yarn/bin/yarn.js
NVM_CD_FLAGS=-q
npm_package_author_name=<REDACTED>
npm_config_save_prefix=^
npm_config_ignore_optional=
HOSTTYPE=x86_64
INIT_CWD=/home/leo/project
```
**Expected behavior**
Use provided `npm_execpath` env var.

**Comment by ayan4m1** [comment]
Please try with the newly published 1.0.4 version.

**Comment by ungarscool1** [comment]
Yup it work, on linux but I have found the same problem on macos
